pkg/apis/externaldns/types.go

```diff
@@ -212,158 +213,159 @@ type Config struct {
-	AWSZoneTagFilter:            []string{},
-	AWSZoneMatchParent:          false,
-	AWSAssumeRole:               "",
-	AWSAssumeRoleExternalID:     "",
-	AWSBatchChangeSize:          1000,
-	AWSBatchChangeSizeBytes:     32000,
-	AWSBatchChangeSizeValues:    1000,
-	AWSBatchChangeInterval:      time.Second,
-	AWSEvaluateTargetHealth:     true,
-	AWSAPIRetries:               3,
-	AWSPreferCNAME:              false,
-	AWSZoneCacheDuration:        0 * time.Second,
-	AWSSDServiceCleanup:         false,
-	AWSSDCreateTag:              map[string]string{},
-	AWSDynamoDBRegion:           "",
-	AWSDynamoDBTable:            "external-dns",
-	AzureConfigFile:             "/etc/kubernetes/azure.json",
-	AzureResourceGroup:          "",
-	AzureSubscriptionID:         "",
-	AzureZonesCacheDuration:     0 * time.Second,
-	CloudflareProxied:           false,
-	CloudflareDNSRecordsPerPage: 100,
-	CloudflareRegionKey:         "earth",
-	CoreDNSPrefix:               "/skydns/",
-	AkamaiServiceConsumerDomain: "",
-	AkamaiClientToken:           "",
-	AkamaiClientSecret:          "",
-	AkamaiAccessToken:           "",
-	AkamaiEdgercSection:         "",
-	AkamaiEdgercPath:            "",
-	OCIConfigFile:               "/etc/kubernetes/oci.yaml",
-	OCIZoneScope:                "GLOBAL",
-	OCIZoneCacheDuration:        0 * time.Second,
-	InMemoryZones:               []string{},
-	OVHEndpoint:                 "ovh-eu",
-	OVHApiRateLimit:             20,
-	PDNSServer:                  "http://localhost:8081",
-	PDNSServerID:                "localhost",
-	PDNSAPIKey:                  "",
-	PDNSSkipTLSVerify:           false,
-	PodSourceDomain:             "",
-	TLSCA:                       "",
-	TLSClientCert:               "",
-	TLSClientCertKey:            "",
-	Policy:                      "sync",
-	Registry:                    "txt",
-	TXTOwnerID:                  "default",
-	TXTPrefix:                   "",
-	TXTSuffix:                   "",
-	TXTCacheInterval:            0,
-	TXTWildcardReplacement:      "",
-	MinEventSyncInterval:        5 * time.Second,
-	TXTEncryptEnabled:           false,
-	TXTEncryptAESKey:            "",
-	TXTNewFormatOnly:            false,
-	Interval:                    time.Minute,
-	Once:                        false,
-	DryRun:                      false,
-	UpdateEvents:                false,
-	LogFormat:                   "text",
-	MetricsAddress:              ":7979",
-	LogLevel:                    logrus.InfoLevel.String(),
-	ExoscaleAPIEnvironment:      "api",
-	ExoscaleAPIZone:             "ch-gva-2",
-	ExoscaleAPIKey:              "",
-	ExoscaleAPISecret:           "",
-	CRDSourceAPIVersion:         "externaldns.k8s.io/v1alpha1",
-	CRDSourceKind:               "DNSEndpoint",
-	ServiceTypeFilter:           []string{},
-	CFAPIEndpoint:               "",
-	CFUsername:                  "",
-	CFPassword:                  "",
-	RFC2136Host:                 "",
-	RFC2136Port:                 0,
-	RFC2136Zone:                 []string{},
```

Comment:
For visibility:
This was changed

RFC2136Host: "", -> RFC2136Host: []string{""},